src/meili/routing.cc

```diff
@@ -15,12 +15,11 @@ namespace valhalla {
 
 namespace meili {
 
-LabelSet::LabelSet(const float max_cost, const float bucket_size) {
+LabelSet::LabelSet(const float max_dist, const float bucket_size) {
```

Comment:
i only changed it because on one of the commits i was still passing in time and it seemed odd to name one cost and one time. so i named them both what they were, dist and time. now that its just dist again i can rename it back to cost.